### ISSUE TITLE
Add table of contents "slugify mode" option for Bitbucket

### ISSUE THREAD

**Issue by alexb148** [yzhang-gh/vscode-markdown#870]
## Proposal
The setting `markdown.extension.toc.slugifyMode` includes table of contents formatting options for Github, Azure, Gitlab and a few others, but not Bitbucket.

Bitbucket links are almost identical to Github links, but are simply prefixed with `markdown-header-`

i.e. they are formatted like so:
`#markdown-header-1-introduction`

Alternatively, just the ability to specify a custom prefix to ToC slugs would solve the issue.

Of course, the ability to specify our own custom format would be amazing, and would solve all cases!

Thanks

**Comment by yzhang-gh** [comment]
I guess you are talking about Bitbucket Cloud? Please check out the [dev build](https://github.com/yzhang-gh/vscode-markdown#latest-development-build). (duplicate of #861)

**Comment by decoder247** [comment]
Yes please to this! I have to manually prepend the "markdown-header-" tag each time before being able to commit changes to the readme at work. Any chance this can be a feature in the production build (i.e. as an option in the slugify settings dropdown)?

**Comment by yzhang-gh** [comment]
Thanks. It will be shipped with the next release, and I'm sure it won't take a month 😅.
